content/guides/user-guides/csi/ceph csi/_index.md

```diff
@@ -0,0 +1,364 @@
+      volumeDevices:
+        - name: data
+          devicePath: /dev/xvda
+  volumes:
+    - name: data
+      persistentVolumeClaim:
+        claimName: raw-block-pvc
+```
+
+# SODA Integration with Kubernetes via SODA CSI Plugin
+
+## Prerequisite
+
+### Ubuntu
+
+version information:
+
+```bash
+root@proxy:~# cat /etc/issue
+Ubuntu 16.04.2 LTS \n \l
+```
+
+### Environment packages
+
+```bash
+apt-get update && apt-get install -y gcc make libc6-dev
+```
+
+### Install Go (v1.12.1)
+
+```bash
+wget https://storage.googleapis.com/golang/go1.12.1.linux-amd64.tar.gz
+tar -C /usr/local -zxvf go1.12.1.linux-amd64.tar.gz
+echo 'export PATH=$PATH:/usr/local/go/bin' >> /etc/profile
+echo 'export GOPATH=$HOME/gopath' >> /etc/profile
+source /etc/profile
+```
+
+### Docker
+
+version information:
+
+```bash
+root@ubuntu:~# docker version
+Client:
+ Version:           18.06.1-ce
+ API version:       1.38
+ Go version:        go1.10.3
+ Git commit:        e68fc7a
+ Built:             Tue Aug 21 17:24:56 2018
+ OS/Arch:           linux/amd64
+ Experimental:      false
+
+Server:
+ Engine:
+  Version:          18.06.1-ce
+  API version:      1.38 (minimum version 1.12)
+  Go version:       go1.10.3
+  Git commit:       e68fc7a
+  Built:            Tue Aug 21 17:23:21 2018
+  OS/Arch:          linux/amd64
+  Experimental:     false
+```
+
+You can install docker using these commands below:
+
+```bash
+wget https://download.docker.com/linux/ubuntu/dists/xenial/pool/stable/amd64/docker-ce_18.06.1~ce~3-0~ubuntu_amd64.deb
+dpkg -i docker-ce_18.06.1~ce~3-0~ubuntu_amd64.deb
+```
+
+### [Kubernetes](https://github.com/kubernetes/kubernetes) local cluster
+
+You can startup `v1.14.0` k8s local cluster by executing commands blow:
+
+```bash
+cd $HOME && git clone https://github.com/kubernetes/kubernetes.git
+cd $HOME/kubernetes && git checkout v1.14.0
+make
+echo alias kubectl='$HOME/kubernetes/cluster/kubectl.sh' >> /etc/profile
+hack/install-etcd.sh
+ENABLE_DAEMON=true ALLOW_PRIVILEGED=true FEATURE_GATES=VolumeSnapshotDataSource=true RUNTIME_CONFIG="storage.k8s.io/v1alpha1=true" LOG_LEVEL=5 hack/local-up-cluster.sh -O
+```
+
+### [OpenSDS](https://github.com/sodafoundation/opensds) local cluster
+
+For testing purposes you can deploy OpenSDS refering
+to [OpenSDS Cluster Installation through Ansible](https://github.com/sodafoundation/opensds/wiki/OpenSDS-Cluster-Installation-through-Ansible)
+.
+
+## Testing steps
+
+##### Following are some additional steps we need to perform for CEPH-RBD CSI Plugin.
+
+* Create a block image/volume inside ceph cluster.
+
+```
+rbd create osdsrbd/vol1 --size 100 --order 12
+```
+
+* Mount a block image.
+
+```
+rbd map osdsrbd/vol1 --id admin
+```
+
+* Adjust the tunables on the ceph cluster.
+
+```
+ceph osd crush tunables hammer
+
+grep -q "^rbd default features" /etc/ceph/ceph.conf || sed -i '/\[global\]/arbd default features = 1' /etc/ceph/ceph.conf
+```
+
+* Change the workplace
+
+  ```
+  cd /opt/opensds-sushi-linux-amd64
```

Comment:
Since its a single line of code please use `cd /opt/opensds-sushi-linux-amd64` . Adding a multi line code block is adding an extra blank box as shown.
![image](https://user-images.githubusercontent.com/19162717/116509087-5066ff00-a8e0-11eb-87ff-93220087a42e.png)

Reply:
@anvithks I have done the changes, please check.